assets/content/static/footer.html

```diff
@@ -8,42 +8,58 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
                 </div>
                 <div class="col-lg-6 text-lg-center btn-wrapper">
                     <a target="_blank" href="https://twitter.com/goasksef"
+
```

Comment:
Can you remove the changes on this file because those are not related to the issue?

Reply:
> Can you remove the changes on this file because those are not related to the issue?

okay sure